addons/mail/wizard/mail_resend_message.py

```diff
@@ -2,27 +2,28 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, fields, models, _, Command
-from odoo.exceptions import UserError
+from odoo.exceptions import AccessError, UserError
+from odoo.tools.mail import email_normalize
 
 
 class MailResendMessage(models.TransientModel):
     _name = 'mail.resend.message'
     _description = 'Email resend wizard'
 
     mail_message_id = fields.Many2one('mail.message', 'Message', readonly=True)
-    partner_ids = fields.One2many('mail.resend.partner', 'resend_wizard_id', string='Recipients')
+    contact_ids = fields.One2many('mail.resend.contact', 'resend_wizard_id', string='Recipients')
     notification_ids = fields.Many2many('mail.notification', string='Notifications', readonly=True)
     can_cancel = fields.Boolean(compute='_compute_can_cancel')
     can_resend = fields.Boolean(compute='_compute_can_resend')
     partner_readonly = fields.Boolean(compute='_compute_partner_readonly')
 
-    @api.depends("partner_ids")
+    @api.depends('contact_ids')
     def _compute_can_cancel(self):
-        self.can_cancel = self.partner_ids.filtered(lambda p: not p.resend)
+        self.can_cancel = self.contact_ids.filtered(lambda p: not p.resend)
 
-    @api.depends('partner_ids.resend')
+    @api.depends('contact_ids.resend')
     def _compute_can_resend(self):
-        self.can_resend = any([partner.resend for partner in self.partner_ids])
+        self.can_resend = any([partner.resend for partner in self.contact_ids])
```

Comment:
while you are changing this line :) 

```suggestion
        self.can_resend = any(partner.resend for partner in self.contact_ids)
```